doc-template/examples/chart.rs

```diff
@@ -1,12 +1,12 @@
 use plotters::prelude::*;
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     let root = BitMapBackend::new("plotters-doc-data/5.png", (640, 480)).into_drawing_area();
-    root.fill(&White);
+    root.fill(&WHITE);
```

Comment:
Seems you already done this. 
Since we use rustdoc as README.md as well. So `README.md` and RustDoc in `lib.rs` are generated by `doc-template/update_readme.sh`. 
If you haven't done this I will do it before merge :)

Reply:
I have no used the scripts since I wasnt sure how to exactly use them. I simply did a project wide replacement of the colors.